packages/core/.yo-rc.json

```diff
@@ -1,5 +1,8 @@
 {
   "generator-theia": {
-    "testSupport": true
+    "testSupport": true,
+    "internalizationSupport": true,
+    "i18nSrcFolder": "src/common/i18n",
```

Comment:
It would be nice to have good defaults for all our extensions, and assume that internationalization is used always resp. infer from whether a source messages file exists. 
I think a top level folder, next to the `src` folder would be appropriate, e.g. `messages` or `i18n`.